### DISCUSSION TITLE
Teleport with self signed certificate

### DISCUSSION THREAD

**Discussion by kalhori124** [gravitational/teleport#7767] [Q&A]
Hi

Is it possible to install Teleport with self signed certificate or Microsoft CA server and bypass ACME during the installation.

Thanks,

**Comment by webvictim** [comment]
Yes, Teleport's default behaviour is to provision a self-signed certificate for the web UI/proxy service if ACME is not enabled during installation.

**Reply by kalhori124** [reply]
Thank you for your help but I am confused, how can I disable ACME and use my own certificate?

Should I skip this **"""** teleport configure --acme --acme-email=your-email@example.com --cluster-name=tele.example.com -o file_ **"""** before teleport start.

When I skipped above step I got three warning lines but I can't open Teleport login page.

WARN [AUTH:1]    Configuration setting auth_service/advertise_ip is not set. guessing 192.168.146.200:3025. service/service.go:1337
WARN             No TLS Keys provided, using self-signed certificate. service/service.go:3305
WARN             Generating self-signed key and cert to /var/lib/teleport/webproxy_key.pem /var/lib/teleport/webproxy_cert.pem. service/service.go:3323

Thanks

**Reply by webvictim** [reply]
Just remove `--acme` and `--acme-email` from the command line, then write that config file to `/etc/teleport.yaml` and restart Teleport.

If it still doesn't work, please paste your `/etc/teleport.yaml` file here.

**Reply by kalhori124** [reply]
I removed both --acme and --acme-email but unfortunately it does not work.

#teleport configure  --cluster-name=tele.example.com -o file
-----------

teleport:
  nodename: sshserver.yaas.bm
  data_dir: /var/lib/teleport
  log:
    output: stderr
    severity: INFO
    format:
      output: text
  ca_pin: ""
auth_service:
  enabled: "yes"
  listen_addr: 0.0.0.0:3025
  cluster_name: sshserver.yaas.bm
ssh_service:
  enabled: "yes"
  labels:
    env: example
  commands:
  - name: hostname
    command: [hostname]
    period: 1m0s
proxy_service:
  enabled: "yes"
  listen_addr: 0.0.0.0:3023
  https_keypairs: []
  acme: {}

Thanks,

**Reply by webvictim** [reply]
What error are you getting when trying to start Teleport?
